language-modelling/language-modelling-student-handbook.md

```diff
@@ -151,7 +151,7 @@ Notice how the code is hidden from view — we can still run it though! Running
 <aside>
 ❗ Stop when you reach the second cell — it requires you to paste a token in to log in to a platform called HuggingFace, the state of the art for storing models + datasets!
 
-The token for our shared account is here (and also at the top of the page): `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`, which you can paste here:
+The token for our shared account is here (and also at the top of the page): `<hugging-face-token>`, which you can paste here:
```

Comment:
Could we add this to the facilitator notes somewhere (that one needs to be created etc.)? And/or add instructions for how students can do it if they're happy to make an account?

Or a list of all accounts needed and how to set them up actually (ChatGPT, Google for Colab, HuggingFace?)

Reply:
yup, that's something i definitely need to get on top of! will add to my todos to fill that all out properly 👍

Reply:
see #46